src/components/navbar/Nav.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+import Navbar from 'react-bootstrap/Navbar';
+import './index.css';
+
+const Nav = () => (
+  <Navbar className="navbar">
+    <Container>
+      <Navbar.Brand href="/">
+        <h2 className="title">COUNTRYPEDIA</h2>
+      </Navbar.Brand>
+    </Container>
+  </Navbar>
```

Comment:
- [ ] The navbar for the country details page suppose to show the country name instead of the `App name` and also have a `back button` in the shape of an arrow when clicked it returns to the home page. So please Fix that.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/217869311-47158078-4073-4bae-b884-8e689866d741.png)  | ![image](https://user-images.githubusercontent.com/68796232/217869219-5d103ef9-726c-4af9-abd1-b73b92a52074.png) |

[Hint] You can do that using conditional rendering see this [example](https://github.com/Omar-Muhamad/Covid-Stats-Capstone/blob/main/src/Components/Header.js) to understand more.